src/opening-period/time-span/TimeSpan.scss

```diff
@@ -0,0 +1,45 @@
+@import 'node_modules/hds-design-tokens/lib/all.scss';
+
+.remove-time-span-button {
+  color: var(--color-coat-of-arms);
+}
+
+.start-and-end-time-container {
+  display: flex;
+  align-items: flex-end;
+}
+
+.dash-between-start-and-end-time-container {
+  height: 56px;
+  display: flex;
+  margin-left: 10px;
+  margin-right: 10px;
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
+}
+
+.time-span-state-container {
+  width: 307px;
+}
+
+.time-span-state {
+  font-family: HelsinkiGrotesk-Regular, Arial, sans-serif;
+}
+
+.time-span-description-container {
+  width: 386px;
+}
+
+.time-span-description {
+  font-family: HelsinkiGrotesk-Regular, Arial, sans-serif;
+}
+
+.time-span-first-header-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.time-span-container {
+  border-top: $spacing-4-xs solid var(--color-coat-of-arms);
+  margin-top: 46px;
```

Comment:
Could this be replaced with some predefined HDS-spacing?

Reply:
There's $spacing-2-xl which is 48px. It's close, I'll check if we could apply that!

- [ ] Check if we can apply $spacing-2-xl instead of hard-coding

Reply:
$spacing-2-xl was close enough, and I checked that the UI also seemed to look ok with it, so applied that!